query-engine/connectors/query-connector/src/filter/scalar.rs

```diff
@@ -58,7 +58,7 @@ impl ScalarFilter {
 
     /// If `true`, the filter can be split into smaller filters executed in
```

Comment:
Comment could be changed to reflect the new naming.

Reply:
good call; done!